lib/l10n/app_en.arb

```diff
@@ -192,7 +192,7 @@
     "@map_page_loading_shops_hint1": {
         "description": "Hint shown nearby the progress bar on the map page. The hint explains what is being loaded."
     },
-    "map_page_loading_shops_hint2": "Still loading stores - seems the area has a big number of organizations",
+    "map_page_loading_shops_hint2": "Still loading stores - seems the area has a big number of them",
```

Comment:
not sure about this, i think i would be better if we just used "stores"

"Still loading stores - seems there are a lot of stores in this area"